src/Component/legacy/src/Exception/StorageUnavailableException.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Resource\Exception;
```

Comment:
If for some reason legacy exception needs to be created, then it should become class alias, otherwise please move it to `src/Component/src/Exception/StorageUnavailableException`